app/src/main/res/layout/item_audio_create_comment.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do you reformat code all file xml ? If you done, I m very sorry !

Reply:
looks like code formation in this file is ok :v